nextcloud_update.sh

```diff
@@ -309,7 +309,22 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    occ_command app:update --all >> Updated_Apps
+fi
+
+# Check which apps got updated
+if [ -n $(occ_command app:update --all) ]
```

Comment:
Better to put everything in one place, so execute `occ_command app:update --all >> Updated_Apps` here instead - in the first `if` statement

Reply:
this is done